src/vs/workbench/browser/parts/menubar/menubarPart.ts

```diff
@@ -106,15 +106,7 @@ export class MenubarPart extends Part {
 	private _modifierKeyStatus: IModifierKeyStatus;
```

Comment:
Changes in this file make sense, mostly simplification and updating of css rules.